components/org.wso2.carbon.identity.conditional.auth.functions.choreo/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/choreo/CallChoreoFunctionImpl.java

```diff
@@ -35,6 +36,9 @@
 import org.wso2.carbon.identity.application.authentication.framework.exception.FrameworkException;
 import org.wso2.carbon.identity.conditional.auth.functions.choreo.internal.ChoreoFunctionServiceHolder;
 import org.wso2.carbon.identity.conditional.auth.functions.common.utils.Constants;
+import org.wso2.carbon.identity.secret.mgt.core.exception.SecretManagementException;
+import org.wso2.carbon.identity.secret.mgt.core.model.ResolvedSecret;
+
 
 import java.io.IOException;
```

Comment:
Shall we remove the extra blank line here?

Reply:
Addressed with commit [97b8dc8](https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/85/commits/97b8dc822153f28e50e8f878d46d78b6d6dfd1c0)